Geometry_Engine/Create/BoundingBox.cs

```diff
@@ -51,6 +53,19 @@ public static BoundingBox BoundingBox(Point centre, Vector extent)
             };
         }
 
+        /***************************************************/
+
+        [Description("Create a BoundingBox using the properties of a cuboid.")]
```

Comment:
Maybe worth mentioning that the bounding box will have its centre in the origin of the global coordinate system?